sdk/core/core/README.md

```diff
@@ -50,10 +56,81 @@ typedef struct {
 } az_span;
 ```
 
-## 3. Strings
+### Strings
 
 A string is a span of UTF-8 characters. It's not a zero-terminated string. Defined in [inc/az_str.h](inc/az_str.h).
 
 ```c
 az_const_span const hello_world = AZ_SPAN_FROM_STR("Hello world!");
 ```
+
+## Examples
+
+TODO
+
+## Troubleshooting
+
+### General
+
+TODO
+
+### Retry policy
+
+While working with Azure you might encounter transient failures. For information about handling these types of failures, see [Retry pattern][azure_pattern_retry] in the Cloud Design Patterns guide, and the related [Circuit Breaker pattern][azure_pattern_circuit_breaker].
+
+## Next steps
+
+### More sample code
+
+TODO
+
+### Additional documentation
+
+TODO
+
+## Contributing
```

Comment:
I assume that this is all boilerplate and identical to our other SDK repos. I have no opinion on this part of the readme.

Reply:
Correct